kube_deploy/Prod/deploy.yml

```diff
@@ -84,6 +84,21 @@ spec:
                 secretKeyRef:
                   name: cica-web-secrets
                   key: cw_session_duration
+            - name: CW_GOVUK_CLIENT_ID
+              valueFrom:
+                secretKeyRef:
+                  name: cica-web-secrets
+                  key: cw_govuk_client_id
+            - name: CW_GOVUK_ISSUER_URL
+              valueFrom:
+                secretKeyRef:
+                  name: cica-web-secrets
+                  key: cw_govuk_issuer_url
+            - name: CW_GOVUK_PRIVATE_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: cica-web-secrets
+                  key: cw_govuk_private_key
```

Comment:
Not sure what the behaviour is if these haven't been applied in the environment?